src/insns/acperm_32bit.adoc

```diff
@@ -50,8 +50,10 @@ The common rules are:
 .. Clear <<m_bit>> unless <<x_perm>> is set
 . <<lm_perm>> cannot be set without <<c_perm>> being set
 .. Clear <<lm_perm>> unless <<c_perm>> is set.
-. <<sl_perm>> cannot be set without <<c_perm>> being set
-.. Clear <<sl_perm>> unless <<c_perm>> is set.
+. <<sl_perm>> cannot be set without <<c_perm>> and <<w_perm>> being set
```

Comment:
Can we weaken this one to just "without <c_perm>"? otherwise we prohibit a future "Elevate Store Local"